### DISCUSSION TITLE
是否有更加简单升级的方法

### DISCUSSION THREAD

**Discussion by D-Sketon** [theme-shoka-x/hexo-theme-shokaX#66] [General]
目前我的升级办法是卸载然后重新安装，然后手动修改配置文件，还有更简单的方法吗

**Comment by zkz098** [comment]
npm源可以 ncu -u 然后重新安装就行
配合`_config.shokaX.yml`可以实现不修改配置文件升级
github源只能重新安装，用`_config.shokaX.yml`可以把改配置文件省掉

**Reply by D-Sketon** [reply]
这样 `_images.yml` 和 `assets` 里的文件也要手动修改吧

**Reply by zkz098** [reply]
assets理论上会自动检测`source/_data`目录下是否存在文件，如果存在则与自带assets合并(_data覆盖自带)
_images.yml可以写一个()
